src/app/examples/modal-example/second-embedded-modal-example/second-embedded-modal-example.component.html

```diff
@@ -0,0 +1,4 @@
+<div>
+  <p>Hello from the second embedded example component</p>
+  <button kirby-button (click)="onHidePress($event)">Hide me</button>
```

Comment:
We use the term `select` to cover both `click` and `tap`event across web + {N}:
```suggestion
  <button kirby-button (click)="onHideSelect($event)">Hide me</button>
```

OR simply:
```suggestion
  <button kirby-button (click)="hideModal($event)">Hide me</button>
```